### ISSUE TITLE
[bitnami/elasticsearch] Ingress: http and host reversed in the rules section

### ISSUE THREAD

**Issue by huguesn** [bitnami/charts#10107]
### Name and Version

bitnami/elasticsearch 18.0.1

### What steps will reproduce the bug?

To reproduce the bug, you just need to enable the ingress:
```yaml
elasticsearch:
  ingress:
    enabled: true
```

### Are you using any custom parameters or values?

Yes, `elasticsearch.ingress.enabled=true`

### What is the expected behavior?

No error, templates can compile.

### What do you see instead?

With `helm template --debug`, you can see that something is wrong, the yaml is not valid.
```yaml
spec:
  rules:
    - http:                                # "host:" should be here
      host: elasticsearch.myhost.local     # "http:" should be here
        paths:
          - path: /
            ...
```

**Comment by fmulero** [comment]
Thanks @huguesn  for reporting this issue. We are going to review soon all ingress objects but if you would like to contribute by creating a PR to solve the issue, The Bitnami team will be happy to review it and provide feedback. [Here](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md) you can find the contributing guidelines.

**Comment by fmulero** [comment]
Hi @huguesn Could you try version 18.1.0? This issue should be fixed in that version

**Comment by fmulero** [comment]
This issue was fixed in #10253
